src/qml/QFieldCloudScreen.qml

```diff
@@ -180,6 +180,16 @@ Page {
         }
       }
 
+      SearchBar {
```

Comment:
Let's rename SearchBar to QfSearchBar and move it into the imports/Theme folder.

We should use it in two other places ATM (RelationCombobox.qml and ValueMap.qml).